slack_sdk/models/blocks/basic_components.py

```diff
@@ -249,7 +249,9 @@ def to_dict(self, option_type: str = "block") -> dict:  # skipcq: PYL-W0221
         self.validate_json()
         if option_type == "dialog":  # skipcq: PYL-R1705
             return {"label": self.label, "value": self.value}
-        elif option_type == "action":
+        elif option_type == "action" or option_type == "attachment":
```

Comment:
This update is not directly related to the issue #990 (it's ok to remove if others think we should have this in a different PR)